src/main/java/org/jenkinsci/plugins/jx/resources/BuildSyncRunListener.java

```diff
@@ -64,22 +63,13 @@
 public class BuildSyncRunListener extends RunListener<Run> {
     protected static final String[] exposeUrlAnnotations = {"jenkins-x.io/exposeUrl", "fabric8.io/exposeUrl"};
     private static final Logger logger = Logger.getLogger(BuildSyncRunListener.class.getName());
-    private long pollPeriodMs = 1000;
-    private String namespace;
+    private final long pollPeriodMs = 1000;
 
-    private transient Set<Run> runsToPoll = new CopyOnWriteArraySet<>();
```

Comment:
Meaningless for a variable in a nonserializable (and in fact non-XStream-able) class to be `transient`.